docs/source/admin/traffic_ops.rst

```diff
@@ -237,6 +237,11 @@ To upgrade from older Traffic Ops versions, stop the service, use :manpage:`yum(
 
 After this completes, see Guide_ for instructions on running the :program:`postinstall` script. Once the :program:`postinstall` script, has finished, run the following command as the root user (or with :manpage:`sudo(8)`): ``systemctl start traffic_ops`` to start the service.
 
+Upgrading to 6.0
+----------------
+
+As of 6.0, Traffic Ops supports PostgreSQL version 13.2. In order to migrate from the prior PostgreSQL version 9.6, it is recommended to use the `pg_upgrade <https://www.postgresql.org/docs/13/pgupgrade.html>`_ tool.
```

Comment:
Might want to specify that this refers to Apache Traffic Control, since the API has its own versioning

Reply:
Done!